docs-src/src/pages/components/FlashTables/FlashProps.js

```diff
@@ -0,0 +1,164 @@
+              <code>rev-Flash--success</code>
+            </span>
+          </Table.Data>
+        </Table.Row>
+
+        <Table.Row>
+          <Table.Data>
+            <Table.HeaderInline>Name:</Table.HeaderInline> <code>warning</code>
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Type:</Table.HeaderInline> bool
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Default:</Table.HeaderInline> False
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Description:</Table.HeaderInline>{' '}
+            <span>
+              {' '}
+              Prop added for warning flash. Adds the class{' '}
+              <code>rev-Flash--warning</code>
+            </span>
+          </Table.Data>
+        </Table.Row>
+
+        <Table.Row>
+          <Table.Data>
+            <Table.HeaderInline>Name:</Table.HeaderInline> <code>alert</code>
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Type:</Table.HeaderInline> bool
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Default:</Table.HeaderInline> False
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Description:</Table.HeaderInline>{' '}
+            <span>
+              {' '}
+              Prop added for alert flash. Adds the class{' '}
+              <code>rev-Flash--alert</code>
+            </span>
+          </Table.Data>
+        </Table.Row>
+
+        <Table.Row>
+          <Table.Data>
+            <Table.HeaderInline>Name:</Table.HeaderInline>{' '}
+            <code>persistent</code>
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Type:</Table.HeaderInline> bool
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Default:</Table.HeaderInline> False
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Description:</Table.HeaderInline>{' '}
+            <span>
+              {' '}
+              Prop added for persistent flash. Adds the class{' '}
+              <code>rev-Flash--persistent</code>
+            </span>
+          </Table.Data>
+        </Table.Row>
+
+        <Table.Row>
+          <Table.Data>
+            <Table.HeaderInline>Name:</Table.HeaderInline> <code>fade</code>
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Type:</Table.HeaderInline> bool
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Default:</Table.HeaderInline> False
+          </Table.Data>
+          <Table.Data>
+            <Table.HeaderInline>Description:</Table.HeaderInline>{' '}
+            <span>
+              {' '}
+              Prop added for fade flash. Adds the class{' '}
+              <code>rev-Flash--fade</code> And fades out after 5secs
+            </span>
+          </Table.Data>
+        </Table.Row>
+
+        <Table.Row>
```

Comment:
delete this for now. we'll add the prop to the table once the enhancement ticket is worked that adds the dismissible flash

Reply:
done